store/postgres/src/store.rs

```diff
@@ -607,10 +615,40 @@ impl Store {
         operations: Vec<EntityOperation>,
         event_source: EventSource,
     ) -> Result<(), StoreError> {
+        // Keep a count of how many entities have been added/removed. This
+        // crucially depends on the fact that all operations are about one
+        // subgraph, with the possible exception that some might touch
+        // the subgraph of subgraphs
+        let mut count = 0;
+        let mut subgraph = None;
+
         for operation in operations.into_iter() {
-            self.apply_entity_operation(conn, operation, event_source)?;
+            if subgraph.is_none() {
+                subgraph = operation
+                    .subgraph()
+                    .filter(|s| !s.is_meta())
+                    .map(|s| s.clone());
+            } else {
+                // Verify that we only have one non-meta subgraph in operations
+                if let Some(other) = operation.subgraph() {
+                    if !other.is_meta() && operation.subgraph() != subgraph.as_ref() {
+                        panic!(
+                            "applying entity operations to two non-metadata subgraphs {:?} and {}",
```

Comment:
These could both be `{}` if we do `subgraph.to_string()`, I think.

Reply:
`subgraph` is an `Option<SubgraphDeploymentId>`, so there's no `to_string()` for it. But this is also something we should never hit, and if we hit it we have a serious bug somewhere; the only thing that users can do with this error message is cut-and-paste it into an issue, there's nothing they can do to address it.

Reply:
Nevermind!